WordPress/Classes/Utility/NotificationsManager.m

```diff
@@ -50,10 +53,6 @@ @implementation NotificationsManager
 
 + (void)registerForPushNotifications
 {
-#if TARGET_IPHONE_SIMULATOR
```

Comment:
I believe this was there because we were formerly just registering for remote notifications. However, now that we have local notifications, we can register in the simulator.